Cargo.lock

```diff
@@ -1482,6 +1482,17 @@ dependencies = [
  "regex",
 ]
 
+[[package]]
+name = "globwalk"
```

Comment:
Oh oof I forgot tera uses globwalk :( it's extremely slow: https://github.com/rust-lang/docs.rs/pull/861/commits/9c9647ded669057493186245f5dcbc3b248d2783

Reply:
We don't actually use `globwalk` in this PR, we are manually loading files the same way the linked docs.rs PR does.